docs/category/category-schema.yaml

```
@@ -71,3 +71,12 @@ definitions:
       updatedAt:
         type: string
         format: date-time
+  categoriesSubjects:
```

Please, add more detailed name to this schema
```suggestion
  categoriesSubjectsNames:
```